emission/analysis/classification/inference/mode.py

```diff
@@ -79,29 +146,39 @@ def getModeQuery(groundTruthMode):
 
   @staticmethod
   def getSectionQueryWithGroundTruth(groundTruthMode):
-    return {"$and": [{'type': 'move'},
+    # return {"$and": [{'type': 'move'},
+    return {"$and": [{"$or": [{'type': { '$exists' : False }}, {'type' : 'move'}]},        
+                     ModeInferencePipeline.getModeQuery(groundTruthMode)]}
+
+  @staticmethod
+  def getSectionQueryWithGroundTruthUser(groundTruthMode, uuid):
+     return {"$and": [{"$or": [{'type': { '$exists' : False }}, {'type' : 'move'}]},        
                      ModeInferencePipeline.getModeQuery(groundTruthMode)]}
 
   # TODO: Refactor into generic steps and results
   def loadTrainingDataStep(self, sectionQuery, sectionDb = None):
+    #logging.setLevel(50)
     logging.debug("START TRAINING DATA STEP")
     if (sectionDb == None):
       sectionDb = self.Sections
 
     begin = time.time()
-    logging.debug("Section data set size = %s" % sectionDb.find({'type': 'move'}).count())
+    logging.debug("Section data set size = %s" % sectionDb.find({'$or':[{'type':{'$exists': False}}, {'type': 'move'}]}).count())
+    #AAAlogging.debug("Section data set size = %s" % sectionDb.find({'type': 'move'}).count())
+
     duration = time.time() - begin
     logging.debug("Getting dataset size took %s" % (duration))
         
     logging.debug("Querying confirmedSections %s" % (datetime.now()))
     begin = time.time()
-    confirmedSections = sectionDb.find(sectionQuery)
+    confirmedSections = sectionDb.find(sectionQuery).sort('_id', 1)
```

Comment:
I don't know either. After you merged it, it got deleted from my code, so I had to put it back in. I'm finding the same thing this morning with some of my code being reverted to how it was when I first pulled it.